### PR TITLE
qa: re enable LC tests

### PR DESCRIPTION
**Pull request by theanalyst**
These are the tests which depend on timing and often end up failing,
let us disable these for now and enable them once we figure out how to
set these right.

Requires: https://github.com/ceph/s3-tests/pull/181 to be merged

Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins test this please

File: qa/tasks/s3tests.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def run_tests(ctx, config):
     """
     assert isinstance(config, dict)
     testdir = teuthology.get_testdir(ctx)
-    attrs = ["!fails_on_rgw", "!lifecycle"]
+    attrs = ["!fails_on_rgw", "!lifecycle_expiration"]
```
Comment (amitkumar50) [inline qa/tasks/s3tests.py]: `These are the tests which depend on timing and often end up failing`
I believe would be good to add few names of tests in commit message.

`let us disable these for now and enable them once we figure out how to set these right.`
Also can create an tracker for this.

**Comment by theanalyst** [comment]
Enabling all the LC tests with timing again with more relaxed time intervals https://github.com/ceph/s3-tests/pull/185

**Comment by theanalyst** [comment]
running http://pulpito.ceph.com/abhi-2017-09-07_12:31:26-rgw-wip-abhi-lc-distro-basic-smithi/ 
(--filter=s3tests was used, since this patch only touches s3tests)

**Comment by theanalyst** [comment]
running as  
```teuthology-suite --suite-repo git://github.com/theanalyst/ceph --suite-branch lc-time-disable --ceph master --ceph-repo git://github.com/ceph/ceph -s rgw --subset 1/3  -m smithi```

http://pulpito.ceph.com/abhi-2017-09-14_09:55:21-rgw-master-distro-basic-smithi/

**Comment by theanalyst** [comment]
@cbodley the run came out clean with exception of multisite failures since the suite wasn't having the multisite fix that went in master, (and the valgrind failure), if this is ok, we can merge https://github.com/ceph/s3-tests/pull/185 to master, cherry-pick to ceph-master and I'll rework this pr to drop the branch changes

**Comment by theanalyst** [comment]
Needs to go in after ceph/s3-tests#187  gets in

**Comment by oritwas** [comment]
jenkins, test this please

**Comment by theanalyst** [comment]
dropped the wip label, @cbodley I guess this can go in now since the related https://github.com/ceph/s3-tests/pull/187 went in?

**Comment by yuriw** [comment]
[wip-yuri-testing-2017-10-09-1919](https://trello.com/c/2ungJPgu/175-wip-yuri-testing-2017-10-09-1919)

**Comment by yuriw** [comment]
[wip-yuri-testing-2017-10-10-2113](https://trello.com/c/6grUQb1g/176-wip-yuri-testing-2017-10-10-2113)

**Comment by yuriw** [comment]
removed tags per @cbodley see test results above

**Comment by cbodley** [comment]
@theanalyst sorry for the delays here - i forget what the failures were, and it doesn't look like the logs are available anymore. could you do another run when you find time?

**Comment by theanalyst** [comment]
@cbodley iirc a few tests still depend on time, I had essentially increased the timeouts to nearly end of the 10s processing window, though that doesn't guarantee  that the LC process has completed in 10s; it is most likely in a teuthology task. Also there is an attribute called lifecycle_expiration on these so if we turn them back on and find failures, we can still run lifecycle tests and just disable the ones with expiration, since lifecycle tests do test other xml attributes and policies etc. I'll run a suite tomorrow

**Comment by yuriw** [comment]
[wip-yuri3-testing-2018-02-15-1735](https://trello.com/c/E8jcp4b2/264-wip-yuri3-testing-2018-02-15-1735)

**Comment by cbodley** [comment]
jenkins test docs

**Comment by cbodley** [comment]
jenkins test docs

**Pull request merged**